### ISSUE TITLE
Much faster in currency section

### ISSUE THREAD

**Issue by DGBNOOB** [DigiByte-Core/DGBio#82]
![much faster currency](https://github.com/DigiByte-Core/DGBio/assets/136190014/96db0b7d-28e8-43fe-853c-1cd6b91a0907)

Same issue as #80, insert theoretically between segwit and enables, "segwit theoretically enables up to 1066 on chain transactions".

**Comment by DennisPitallano** [comment]
@DGBNOOB 
I've already made adjustments to the sections that impact only the English translations. Now, we'll also require translations for the other languages to synchronize the content.

cheers;

**Comment by DGBNOOB** [comment]
Perfect, my suggestion was going to be to update the English version first as a base for the rest of the translations. @Sbosvk has translation software, once English version is complete, we can then work on having it translated and proofread by native speakers of the other languages.

**Comment by Sbosvk** [comment]
I am working on another PR right now and will get to translating this as well. I still think we need to get rid of some translations simply because it is not viable to have this many, unfortunate as it is. But that is a discussion for another time.

**Comment by DennisPitallano** [comment]
@DGBNOOB 
I have completed the changes for this issue, but they only affect the English version. Should we wait for the other languages to be updated as well, or can I proceed to update them later on?
the same with #80

**Comment by Sbosvk** [comment]
We can update the languages at a later time, I will not have time for some time. I will probably eventually start overhauling the way we actually handle translations so we can remove the necessity for nearly 30 index files and folders.

**Comment by DGBNOOB** [comment]
@DennisPitallano let’s do English translation only.

**Comment by DennisPitallano** [comment]
Hi @DGBNOOB @Sbosvk 
Showing the preview of the change, Could you please confirm if this aligns with our expectations?
![image](https://github.com/DigiByte-Core/DGBio/assets/3394208/c130cddb-bb97-4e5d-88c9-c49828295657)

**Comment by DGBNOOB** [comment]
Looks good
